### ISSUE TITLE
video is not downloaded, exists, but it is recorded on the archive! `Not Available.mhtml has already been downloaded`

### ISSUE THREAD

**Issue by estatistics** [yt-dlp/yt-dlp#9283]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting a bug unrelated to a specific site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

Video exists on youtube (H7BFCfmKQUE), it is recorded in archive without to be downloaded in real! 

I have many youtube channels that i have archived. Now, i must check all channels, if video exists or not and if they have been recorded in the archive. 

I think archive must be splitted to parts. 
FILES that have been in real downloaded in DISK by checking whatever
CODES that are private / not found with a tag before them at least for youtube 
and at best these fields can be customizable maybe for different webs. 

youtube fddf2323ds

NOT FOUND gd665sdgfsd
PRIVATE        fdf4sfd5dssd
NOT DOWNLOADED fdfd23342
etc... 
and for future uses
DONTDOWNLOAD ff323easd 

Now, i try to run (ll.txt contains all yt codes that must be downloaded) to find what files have been downloaded in actuality. 
```
LN="ll.txt";    while IFS='' read -r LINE || [ -n "${LINE}"]; do
if tree -i | grep -- "${LINE}" 1>/dev/null; then echo "YT_Downloaded" 1>/dev/null; else echo "NOTDOWNLOADED" "${LINE}" ; fi;   done < "${LN}" 
```

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
yt-dlp -vU --cookies-from-browser firefox --limit-rate 2800k --write-description --embed-thumbnail --embed-metadata --download-archive archive.txt -o "%(uploader)s/Loose Videos/%(title)s.%(ext)s" H7BFCfmKQUE
[debug] Command-line config: ['-vU', '--cookies-from-browser', 'firefox', '--limit-rate', '2800k', '--write-description', '--embed-thumbnail', '--embed-metadata', '--download-archive', 'archive.txt', '-o', '%(uploader)s/Loose Videos/%(title)s.%(ext)s', 'H7BFCfmKQUE']
Extracting cookies from firefox
[debug] Extracting cookies from: "/home/elias/.mozilla/firefox/l9alz29x.default-release/cookies.sqlite"
Extracted 34 cookies from firefox
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.12.30 from yt-dlp/yt-dlp [f10589e34] (zip)
[debug] Python 3.11.8 (CPython x86_64 64bit) - Linux-6.1.0-15-amd64-x86_64-with-glibc2.37 (OpenSSL 3.1.5 30 Jan 2024, glibc 2.37)
[debug] exe versions: ffmpeg 6.1.1-1 (setts), ffprobe 6.1.1-1, rtmpdump 2.4
[debug] Optional libraries: Cryptodome-3.11.0, brotli-1.1.0, certifi-2023.11.17, mutagen-1.46.0, pyxattr-0.8.1, requests-2.31.0, sqlite3-3.45.1, urllib3-1.26.18, websockets-10.4
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests
[debug] Extractor Plugins: oauth2 (YoutubeBaseInfoExtractor), oauth2 (YoutubeClipIE), oauth2 (YoutubeConsentRedirectIE), oauth2 (YoutubeFavouritesIE), oauth2 (YoutubeIE), oauth2 (YoutubeMusicSearchURLIE), oauth2 (YoutubeNotificationsIE), oauth2 (YoutubeSearchDateIE), oauth2 (YoutubeSearchIE), oauth2 (YoutubeSearchURLIE), oauth2 (YoutubeTabBaseInfoExtractor), oauth2 (YoutubeTabIE)
[debug] Plugin directories: ['/home/elias/.local/lib/python3.11/site-packages/yt_dlp_plugins']
[debug] Loaded 1798 extractors
[debug] Loading archive file 'archive.txt'
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: stable@2023.12.30 from yt-dlp/yt-dlp
yt-dlp is up to date (stable@2023.12.30 from yt-dlp/yt-dlp)
[youtube+oauth2] Extracting URL: H7BFCfmKQUE
[youtube+oauth2] H7BFCfmKQUE: Downloading webpage
[youtube+oauth2] H7BFCfmKQUE: Downloading ios player API JSON
WARNING: [youtube+oauth2] Skipping player response from ios client (got player response for video "M5t4UHllkUM" instead of "H7BFCfmKQUE"); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
[youtube+oauth2] H7BFCfmKQUE: Downloading android player API JSON
WARNING: [youtube+oauth2] Skipping player response from android client (got player response for video "M5t4UHllkUM" instead of "H7BFCfmKQUE"); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: [youtube+oauth2] Skipping player response from web client (got player response for video "M5t4UHllkUM" instead of "H7BFCfmKQUE"); please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: Only images are available for download. use --list-formats to see them
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, size, br, asr, vext, aext, hasaud, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] H7BFCfmKQUE: Downloading 1 format(s): sb0
[info] Writing video description to: TUS/Loose Videos/Video Not Available.description
[info] Downloading video thumbnail 43 ...
[info] Video Thumbnail 43 does not exist
Deleting existing file TUS/Loose Videos/Video Not Available.jpg
[info] Downloading video thumbnail 42 ...
[info] Writing video thumbnail 42 to: TUS/Loose Videos/Video Not Available.jpg
[debug] Invoking mhtml downloader on "https://i.ytimg.com/sb/M5t4UHllkUM/storyboard3_L2/M$M.jpg?sqp=-oaymwENSDfyq4qpAwVwAcABBqLzl_8DBgi867OjBg==&sigh=rs$AOn4CLD6zz6wUMPtwBuO9zjkyt-TR6iF_g"
[download] TUS/Loose Videos/Video Not Available.mhtml has already been downloaded
[download] 100% of   20.07KiB
[Metadata] Skipping images
[EmbedThumbnail] Skipping images
[debug] Adding to archive: youtube H7BFCfmKQUE
```

**Comment by estatistics** [comment]
forgot to use oauth2. Now they are downloading.  BUT the problem with recording in archive videos that have not downloaded in real remains! 

`yt-dlp --username oauth2 --password ''  --limit-rate 800k --write-description --embed-thumbnail --embed-metadata --download-archive archive.txt -o "%(uploader)s/Loose Videos/%(title)s.%(ext)s" H7BFCfmKQUE`

**Comment by bashonly** [comment]
do this:
> - [x] I've verified that I have updated yt-dlp to nightly or master ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))

this was fixed by d63eae7e7ffb1f3e733e552b9e5e82355bfba214

closing as duplicate of #7715

**Comment by estatistics** [comment]
Yeah, indeed It was downloaded even without master version. I updated to master.  
However, i have some questions after updating. 

`got player response for video "M5t4UHllkUM" instead of "H7BFCfmKQUE"`

In the archive is recorded H7BFCfmKQUE instead of M5t4UHllkUM.
When someone would like to double check if archive is accurate between archive and written files  with id in their filename, then this is not the ideal situation. 

Cause i have in archive H7BFCfmKQUE but in disk H7BFCfmKQUE. 
So, you must employ different "Tags" in archive based on the status
eg. 

youtube archive  
Changed  H7BFCfmKQUE
Current  M5t4UHllkUM
 etc. 

and why browser plays H7BFCfmKQUE while M5t4UHllkUM is unavailable?
